exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsExporter.java

```diff
@@ -52,7 +53,8 @@
 
   private static final Object monitor = new Object();
 
-  private final StackdriverExporterWorkerThread workerThread;
+  private final StackdriverExporterWorker worker;
```

Comment:
nit: this can be removed, having a reference to the daemon thread should be enough.